{{ cookiecutter.repo_name }}/Dockerfile

```diff
@@ -1,4 +1,8 @@
-FROM prefecthq/prefect:{{ cookiecutter.prefect_docker_image_tag }}
+FROM prefecthq/prefect:2.5.0-python3.9
 COPY requirements.txt .
+COPY add_nexus.sh .
+RUN chmod +x ./add_nexus.sh
+RUN ./add_nexus.sh
 RUN pip install -r requirements.txt
```

Comment:
Kan je ook zoiets doen?:
> pip3 install -r requirements.txt \
>     --extra-index-url http://do-prd-mvn-01.do.viaa.be:8081/repository/pypi-all/simple \
>     --trusted-host do-prd-mvn-01.do.viaa.be

Dan is die nexus.sh en de bovenstaande commando's ook overbodig.

Reply:
Inderdaad gemakkelijker, ik was even vergeten dat die flags een mogelijkheid waren.